doc/en/components/layouts/dock-manager.md

```diff
@@ -294,6 +294,34 @@ this.dockManager.addEventListener('activePaneChanged', ev => {
 });
 ```
 
+### Docking
+
+When you start dragging a floating pane, a different docking indicators will appear depending on the position of the dragged pane. There are four main types of docking - root docking, pane docking, document host docking and splitter docking.
+
+#### Root Docking
+
+In this type of docking, the dragged pane will become a child of the Dock Manager `RootPane`. Depending on the root docking indicator position, the orientation and other properties of the root pane, the existing root pane will be modified or will be replaced with a newly created one.
+
+<img class="responsive-img" src="../../images/dockmanager-root-docking.jpg"/>
+
+#### Pane Docking
+
+Docking indicators will appear in the center of a content pane or tab group pane when dragging the floating pane over it. Depending on the docking indicator position, the orientation and other properties of the target pane, the dragged pane will either become a child of the target pane's parent split pane, the target pane will be replaced with a newly created split pane or a tab group pane will be created.
```

Comment:
"Depending on the docking indicator position, the orientation and other properties of the target pane, the dragged pane will either become a child of the target pane's parent split pane, the target pane will be replaced with a newly created split pane or a tab group pane will be created."
This whole statement doesn't correspond to docking into a CP/TGP.